### PR TITLE
[Android] better colour for inactive lane icons

### PR DESCRIPTION
**Pull request by RedAuburn**

### CODE REVIEW

File: android/app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -56,8 +56,8 @@
   <color name="icon_tint">#FF757575</color>
   <color name="icon_tint_night">#FFC4C6C7</color>
 
-  <color name="icon_tint_light">#FFBDBDBD</color>
-  <color name="icon_tint_light_night">#FF76797C</color>
+  <color name="icon_tint_light">#FF007CCB</color>
+  <color name="icon_tint_light_night">#FF2C88AD</color>
```
Comment (biodranik) [inline android/app/src/main/res/values/colors.xml:60]: iconTintLight is used in different places, not only for lanes. Won't this change break these other places? If yes, then lanes tint/color should be introduced as a separate constant.

**Comment by RedAuburn** [comment]
now using new colour definitions, although i can't test the android auto version (why are there two implementations for displaying turn lanes??)

File: android/app/src/main/res/values/colors.xml (hunk 1 above)
Comment (RedAuburn) [reply android/app/src/main/res/values/colors.xml]: fixed

**Comment by biodranik** [comment]
> now using new colour definitions, although i can't test the android auto version (why are there two implementations for displaying turn lanes??)

@AndrewShkrob are lanes displayed in AA?

**Comment by RedAuburn** [comment]
Finally managed to get the AAuto DHU to work:
Screenshot from 2024-08-03 18-07-24
fixing now.

**Comment by RedAuburn** [comment]
fixed colours for Android Auto
bitmap

**Review by biodranik — APPROVED** [review]
Thanks a lot!

**Pull request merged**